runtime-builder/go-1.8.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
+- name: 'gcr.io/${PROJECT_ID}/go1-builder:<BUILD_TAG>'
```

Comment:
s/${PROJECT_ID}/gcp-runtimes/
s/<BUILD_TAG>/1.8/

This file will be used directly as the input for the publishing system to produce a published version of this.  As Nick mentioned, the system will know how to replace 1.8 with the digest tag.